.travis.yml

```diff
@@ -33,7 +33,7 @@ matrix:
       env: PHPSTAN=1 DEFAULT=0
 
 before_install:
-  - if [[ ${TRAVIS_PHP_VERSION} != "7.4snapshot" ]]; then phpenv config-rm xdebug.ini; fi
+  - phpenv config-rm xdebug.ini
```

Comment:
7.4 doesnt have xdebug enabled, does it?

Reply:
Snapshot and nightly builds on Travis do not have any third-party packages, like xdebug. The stable build does include a number of them, like xdebug.